src/js/core.js

```diff
@@ -1,17 +1,17 @@
 
+import { merge, isUndefined, forEach } from 'lodash';
 const fs = require('fs');
 const path = require('path');
 const request = require('request');
 const Mustache = require('mustache');
-const _ = require('lodash');
 
-let allCfg = JSON.parse(fs.readFileSync(path.join(__dirname, 'config.json')));
-let nodeEnv = process.env.NODE_ENV || 'development';
+const allCfg = JSON.parse(fs.readFileSync(path.join(__dirname, 'config.json')));
+const nodeEnv = process.env.NODE_ENV || 'development';
 let cfg = allCfg.APP;
 
-if (!_.isUndefined(allCfg[nodeEnv])) cfg = _.merge(cfg, allCfg[nodeEnv]);
+if (!isUndefined(allCfg[nodeEnv])) cfg = merge(cfg, allCfg[nodeEnv]);
```

Comment:
I actually think this change hurts code readability

Reply:
We are currently using 2 or 3 methods from the lodash library, I don't think we should be importing the entire lodash library for this.

Albeit, I'm a readability first kind of guy, perhaps, we should decide if this is a trade off we are willing to make.